jboss-eap-xp1-openjdk11-openshift.json

```diff
@@ -0,0 +1,123 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-eap-xp1-openjdk11-openshift",
+        "annotations": {
+            "description": "ImageStream definition for JBoss Enterprise Application Platform expansion pack with OpenJDK 11",
+            "openshift.io/provider-display-name": "Red Hat."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap-xp1-openjdk11-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP XP with OpenJDK 11",
```

Comment:
should be "JBoss EAP XP 1.0 with OpenJDK 11"

Reply:
Here, I was trying to normalize out the 1.0, which it will get from the version: "1.0" in the metadata. I'll add it to the display name though too, as thinking about it, in older OCP you can see it in the console etc.